### ISSUE TITLE
Event ID 8000

### ISSUE THREAD

**Issue by red-erik** [lithnet/access-manager#24]
Hello,
my log is full of:

Category: Lithnet.AccessManager.Server.UI.ActiveDirectoryDomainPermissionViewModel
EventId: 8000

Group membership lookup error

Exception: 
System.NullReferenceException: Object reference not set to an instance of an object.
   at Lithnet.AccessManager.Server.UI.ActiveDirectoryDomainPermissionViewModel.IsGroupMember(GroupPrincipal g, SecurityIdentifier userSid, HashSet`1 searchedGroups) in D:\dev\git\lithnet\access-manager\src\Lithnet.AccessManager\Lithnet.AccessManager.Server.UI\ViewModels\ActiveDirectoryDomainPermissionViewModel.cs:line 149
   at Lithnet.AccessManager.Server.UI.ActiveDirectoryDomainPermissionViewModel.IsGroupMember(SecurityIdentifier groupSid, SecurityIdentifier userSid) in D:\dev\git\lithnet\access-manager\src\Lithnet.AccessManager\Lithnet.AccessManager.Server.UI\ViewModels\ActiveDirectoryDomainPermissionViewModel.cs:line 144
   at Lithnet.AccessManager.Server.UI.ActiveDirectoryDomainPermissionViewModel.CheckAcaoStatus() in D:\dev\git\lithnet\access-manager\src\Lithnet.AccessManager\Lithnet.AccessManager.Server.UI\ViewModels\ActiveDirectoryDomainPermissionViewModel.cs:line 74

but everything seems to be up and running fine.
It should be related to "Access Control Assistance Operators Membership" check failing.

Regards,

Red.

**Comment by ryannewington** [comment]
Yep. It's the lack of ACAO group in the domain that's causing this. It's safe to ignore. Technically, the minimum supported version for AMS is Windows 2012, since 2008 is not supported by MS any more, but being aware of your scenario, and your patience in waiting for the solution to multi-forests I built capability behind the scenes for things to work for you. But that's why stuff like the authorisation server overrides we did are done by hand, and not exposed in the UI.
